server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/rfc/model/RequestObject.scala

```diff
@@ -0,0 +1,41 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+
+package org.apache.james.jmap.rfc.model
+
+import eu.timepit.refined.types.string.NonEmptyString
+import org.apache.james.jmap.rfc.model.RequestObject.Capability
+import play.api.libs.json._
+import be.venneborg.refined.play.RefinedJsonFormats._
+
+case class RequestObject(using: Seq[Capability], methodCalls: Seq[Invocation], createdIds: Option[CreatedIds] = None)
+
+object RequestObject {
+
+  case class Capability(value: NonEmptyString)
+
+  implicit val capabilityFormat: Format[Capability] = Json.valueFormat[Capability]
```

Comment:
private?

Reply:
We need to import it in tests

Reply:
no you don't. You want to test the feature of serialization/deserialization not to test implicits.